functions/go/set-labels/transformer/const.go

```diff
@@ -0,0 +1,195 @@
+package transformer
+
+const (
+	fnConfigGroup         = "fn.kpt.dev"
+	fnConfigAPIVersion    = "v1alpha1"
+	fnConfigKind          = "SetLabels"
+	commonLabelFieldSpecs = `
```

Comment:
Does it make more sense to store it as yaml?
@yuwenma Thoughts?

Reply:
+1 to change this. 
I think it is better to use the IsGVK and other KRM resource type specific functions (We can provide some more functions on the SDK side to make it simpler). 

Json/Yaml parsing the resources (even with `json`) set the wrong view on users for KRM resources editing. We expect the functions to change KRM resource by its definition (so relying on OpenApi schema is reasonable), not by the yaml/json raw data (similar as helm values and kustomize vars).